sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1804,6 +1806,10 @@ export interface ContainerRenewLeaseOptions extends CommonOptions {
     conditions?: ModifiedAccessConditions;
 }
 
+// @public
+export interface ContainerRequestConditions extends LeaseAccessConditions, ModificationConditions {
```

Comment:
In some cases you use an intersection, in other cases you use this pattern where you extend multiple interfaces. Is there a reason?

Reply:
I can't recall why I use `extend` here. Will unify to use intersection.